text.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Text</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header class="nav-header">
+        <div class="nav-header-div">
+            <div class="nav-header-div-lt">
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="index.html">Home</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="docs.html">Docs</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="#">About us</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="#">Blogs</a>
+            </div>
+            <div class="nav-header-div-rt">
+                <a href="https://github.com/eakansh125/Fleet-UI/tree/dev"><img class="margin-lr"
+                        src="https://img.icons8.com/ios-filled/20/000000/github.png" /></a>
+                <a href="#"><img class="margin-lr"
+                        src="https://img.icons8.com/material/20/000000/do-not-disturb-2.png" /></a>
+            </div>
+        </div>
+    </header>
+
+    <div class="view">
+        <aside>
+            <div class="nav-pill">
+                <a href="docs.html">Avatar</a>
+                <a href="badge.html">Badge</a>
+                <a href="alert.html">Alert</a>
+                <a href="button.html">Button</a>
+                <a href="card.html">Card</a>
+                <a  href="image.html">Image</a>
+                <a href="input.html">Input</a>
+                <a class="active"href="text.html">Text</a>
+            </div>
+        </aside>
+
+        <main class="main-view">
+            <div class="main-view-container">
+                <h1>Text</h1>
+                <p>Images are provided with a source either from the local system or from the web in different
+                    extensions.</p>
+                <h2>Headings</h2>
+                <p>A responsive image changes its size and adjusts accordingly as the screen size changes</p>
+                <div class="component-container">
+                    <div class="text-contaienr flex-column">
+                        <h1>Sipping on straight chlorine</h1>
```

Comment:
use <p> tags instead of heading tags as they convey meaning. for ex, h1 indicates the main title.

Reply:
These were meant to indicate the headings themselves. But I got your point, will keep a check moving forward! Thanks